src/main/java/me/mrCookieSlime/Slimefun/Events/DisenchanterEvent.java

```diff
@@ -0,0 +1,42 @@
+package me.mrCookieSlime.Slimefun.Events;
+
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.bukkit.inventory.ItemStack;
+
+public class DisenchanterEvent extends Event implements Cancellable {
```

Comment:
I'd rather rename this to "AutoDisenchantEvent", since it represents the "disenchantment", not the "disenchanter" and the machine is also called "Auto-Disenchanter".